packages/valory/skills/decision_maker_abci/behaviours/base.py

```diff
@@ -620,6 +633,11 @@ def payment_token(self) -> str:
         subscription_params = self.subscription_params
         return subscription_params["payment_token"]
 
+    @property
+    def is_xdai(self) -> bool:
+        """Check if the payment token is xDAI"""
+        return self.payment_token == ZERO_ADDRESS
```

Comment:
Can you explain this?

Reply:
When the payment token for the sbuscription is xdai (the native token of the chain), nevermined sets the payment address to the zeroAddress.

Reply:
Could you leave this as a comment in the code?